src/domain/occurrences/occurrencesTable/ActionsDropdown.tsx

```diff
@@ -0,0 +1,239 @@
+import classNames from 'classnames';
+import { IconAngleDown, IconFill, IconPerson } from 'hds-react';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+
+import AlertModal from '../../../common/components/modal/AlertModal';
+import useKeyboardNavigation from '../../../hooks/useDropdownKeyboardNavigation';
+import useLocale from '../../../hooks/useLocale';
+import IconDelete from '../../../icons/IconDelete';
+import { ROUTES } from '../../app/routes/constants';
+import { OccurrenceInTable } from '../types';
+import styles from './actionsDropdown.module.scss';
+
+const MenuItem: React.FC<{
+  isFocused: boolean;
+  onClick: () => void;
+}> = ({ children, isFocused, onClick }) => {
+  const component = React.useRef<HTMLLIElement>(null);
+
+  const onKeyDown = (event: React.KeyboardEvent<HTMLLIElement>) => {
```

Comment:
Can you think of any other keys that would be used here in the future? If not, maybe change the switch case

Reply:
Changed to if statement